utils/lpac/utils.c

```diff
@@ -7,30 +7,19 @@
 #include <stdlib.h>
 #include <string.h>
 
-static bool is_numeric(const char *value) {
-    if (value == NULL)
-        return false;
-    for (size_t i = strlen(value); i > 0; --i) {
-        if (isdigit(value[i]))
-            continue;
-        return false;
-    }
-    return true;
-}
-
 const char *getenv_str_or_default(const char *name, const char *default_value) {
     const char *value = getenv(name);
-    if (value == NULL)
+    if (value == NULL || strlen(value) == 0)
         return default_value;
     return value;
 }
 
 bool getenv_bool_or_default(const char *name, const bool default_value) {
     const char *value = getenv(name);
-    if (value == NULL)
+    if (value == NULL || strlen(value) == 0)
         return default_value;
-    if (is_numeric(value))
-        return strcmp(value, "0") != 0;
+    if (isdigit(value[0]))
```

Comment:
if user uses `ENV=-1`...

Reply:
boolean variable, typical values ​​are generally: `0` or `1`

Reply:
> typical values

Some people like to use unusual values. If we don't like, reject explicitly or document it.

Reply:
Yes, needs defined a boolean type in ENVVARS.md